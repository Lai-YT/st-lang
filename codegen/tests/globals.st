% const with type
const i: int := 0
% var with no type
var j := i
% var with no init expr
var k: int
% var with type int & init expr
var l: int := 1
% const string
const s: string(6) := "string"
% var with type bool & init expr
var b: bool := true
% var with no type bool & init expr
var c := false

k := 10

put i
put j
put k
put l
put s
put b
put c
