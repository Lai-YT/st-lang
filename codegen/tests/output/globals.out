0
0
10
1
string
true
false
